go.mod

```diff
@@ -48,3 +49,5 @@ require (
 	k8s.io/client-go v0.0.0-20180806134042-1f13a808da65
 	k8s.io/kube-openapi v0.0.0-20180731170545-e3762e86a74c // indirect
 )
+
+replace github.com/denisenkom/go-mssqldb => github.com/orenbm/go-mssqldb v0.0.0-20191023140747-b0d280c79777
```

Comment:
This needs to be cyberark/conjur org hosted repo

Reply:
this is probably true, if we want to use it in the actual broker now. let me look into how we can get this going.

Reply:
I moved the content from @orenbm's fork into https://github.com/cyberark/go-mssqldb after reviewing the upstream repo and noting that there has been no activity by the maintainer this month. strong preference to contribute our changes upstream to the project (along with whatever improvements we can help make after leveraging the library ourselves), but for now if we have to use a fork let's use our official one.